offers/README.md

```diff
@@ -246,126 +217,104 @@ fileName:offer.json
         "publisherId":"4ed4b62b-aa4c-43a1-8064-nb7d1368e17a",
-        "startDate":"2018-10-20T12:12:00.000Z",
+        "startDate":"2020-04-20T12:13:13.000Z",
         "type":{
             "name":"discount",
             "value":20
          }
      }'
 ```
 
-You should pass the `brandId` of the Brand you wish to submit the offer to in the URL path and the offer parameters in the payload.
-
-As required parameters, you need to set an offer `name`, your `accountId` as the `publisherId`, a `startDate` at least three weeks from today, the type of offer between `amount` or `discount`, an offer `value`, and the `countryCode` where the offer will be available. The `value` is a percentage if the offer `type` is `discount` or a fixed amount in the local currency of the `countryCode` specified.
+As required parameters, you need to set an offer `name`, your `accountId` as the `publisherId`, the `brandId`, a `startDate`, the type of offer between `amount` or `discount`, an offer `value`, and the `countryCode` where the offer will be available. The `value` is a percentage if the offer `type` is `discount` or a fixed amount in the local currency of the `countryCode` specified.
 
 Check out the [Offer API Reference](https://reference.fidel.uk/v1/reference#create-offer) for more detailed documentation about available Offer API endpoints.
 
 <hr>
 
 ## Offer Lifecycle
 
-### Draft
-First status of an offer after creation. The offer has not been submitted to a publisher. A locations file may have been uploaded for the offer, but no locations would be linked at this stage.
-
-### Submitted
-Offer has been submitted to a publisher for acceptance.
+An offer qualifies transactions depending on a combination of three properties: `startDate, endDate` and `locationsTotal`. On the dashboard offers are grouped between pending, all-set, live and expired.
 
-### Syncing
-Offer has been accepted by the publisher and participating locations are now being onboarded with the schemes.  This stage takes between 1 and 10 business days.
-
-### Active
-Offer goes active on `startDate` and starts qualifying transactions at the participating locations.
-  
-### Rejected
-If an offer is rejected by a publisher, you receive a notification email with the rejection reason and you can to edit the offer parameters and resubmit the offer. 
+If the offer is labeled as pending it means that the `startDate` is in the future. When an offer has at least one location linked to it and the `startDate` is set in the future, the offer is placed in the all-set group. An offer is live when today's date is between the offer's `startDate` and `endDate` and `locationsTotal` is greater than `0`.
 
 ### Pending
-Offer was created with no publisher. These offers are advertised to publishers on the Fidel CLO marketplace.
+Offer `startDate` is set in the future. Offer will start qualifying transactions made at locations linked to the offer after the `startDate`.
```

Comment:
I think some words are missing here, don't really understand this sentence. maybe `Offer will start qualifying when transactions are made`?